### ISSUE TITLE
Is it any reason you did not use layer norm in tabular_nn?

### ISSUE THREAD

**Issue by diff7** [awslabs/autogluon#607]
Hi, I am just curious. 
It seems that layer norm was made for feature-wise normalization and may work a bit better than batch norm. But you use batch norm. Is it any specific reason for it?

Sorry for one more issue but this question did not let me go.

**Comment by Innixma** [comment]
@jwmueller Could you answer this?

**Comment by diff7** [comment]
Hi, got it. Thank you for your reply. 

There is might be a chance that I will be testing different architectures for tabular nns and if I do most likely I will be benchmarking results with your solution. So, I will share results if I find something.

**Comment by diff7** [comment]
I did some tests on a few datasets so far but with different net parameters on each and did not find any  significant difference. 
but my network was written in pytorch, however it resembles your architecture. 

![image](https://user-images.githubusercontent.com/34035501/90244878-4aa4b880-de3a-11ea-8eef-d8886d81fe15.png)

![image](https://user-images.githubusercontent.com/34035501/90245443-5cd32680-de3b-11ea-80bd-93e6bc61089e.png)

**Comment by Innixma** [comment]
Thanks! At the very least, this should mean our NN implementation isn't completely unreasonable. If you end up wrapping your model in AutoGluon's AbstractModel API, I'd be happy to review a PR to add it in, especially if it has some unique property to it that differs from our model such that they could ensemble well together, or if it is able to train faster for example.
